docs_user/modules/con_fips-support.adoc

```diff
@@ -0,0 +1,12 @@
+[id="fips-support_{context}"]
+
+= FIPS support
+
+
+There is a major difference with how FIPS was configured in {OpenStackPreviousInstaller} and how it is enabled and enforced in operator deployments.
+
+In {OpenStackPreviousInstaller} FIPS is enabled as part of it's configuration.
+
+In operator deployments there is no specific FIPS configuration at the operator level. If your deployment has FIPS compliance requirements, the whole {OpenShift} ({OpenShiftShort}) cluster must be installed with FIPS being enabled. This will have happened before proceeding with this guide and installing {OpenStackShort}.
+
```

Comment:
Is it the case that if the source OpenStack deployment is non-FIPS, the destination OpenShift should be configured as non-FIPS too? (If yes then this is probably worth a note?)

Reply:
Thanks for the comment, I'll update it with JP's feedback: "If you are adopting an OSP 17.1 FIPS environment to RHOSO 18.0, your adopted cluster will remain a FIPS cluster (there is no option to change FIPS status during adoption). You must deploy a FIPS OCP cluster to adopt your OSP 17.1 FIPS control plane"